WindowsServerDocs/storage/storage-spaces/delimit-volume-allocation.md

```diff
@@ -48,17 +48,12 @@ The volume goes offline and becomes inaccessible until the servers are recovered
 
 ### New: delimited allocation
 
-With delimited allocation, you specify a subset of servers to use (minimum three for three-way mirror). The volume is divided into slabs that are copied three times, like before, but instead of allocating across every server, **the slabs are allocated only to the subset of servers you specify**.
-
-![Diagram showing the volume being divided into three stacks of slabs and distributed only to three of six servers.](media/delimit-volume-allocation/delimited-allocation.png)
+With delimited allocation, you specify a subset of servers to use (minimum four). The volume is divided into slabs that are copied three times, like before, but instead of allocating across every server, **the slabs are allocated only to the subset of servers you specify**.
 
+For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1,2,3,4.
```

Comment:
This line still needs normal spacing between the numerals:
```suggestion
For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1, 2, 3, 4.
```
@JohanFreelancer9 : Do you have time to suggest any other grammatical or readability improvements in this sentence?

Reply:
@illfated  Looks good to me.